lookout/style/format/feature_extractor.py

```diff
@@ -594,7 +594,7 @@ def _add_noops(self, vnodes: Sequence[VirtualNode], path: str, index_labels: boo
                                                 end=Position(0, 1, 1), y=noop_label, path=path))
         for vnode, next_vnode in zip(vnodes, islice(vnodes, 1, None)):
             augmented_vnodes.append(vnode)
-            if vnode.y is None and next_vnode.y is None:
+            if vnode.y is None and not vnode.is_accumulated_indentation and next_vnode.y is None:
```

Comment:
What if it is accumulated dedentation?

Reply:
sorry I misread your comment. dedentation is also tagged as accumulated_indentation